libs/ts4wing/src/compiler.ts

```diff
@@ -0,0 +1,44 @@
+import { join } from "path";
+
+export interface WingTSOptions {
+  workDir: string;
+  entrypoint: string;
+}
+
+export async function compileTypescriptForWing(options: WingTSOptions) {
```

Comment:
```suggestion
export async function compile(options: WingTSOptions) {
```

I think there's sufficient ambient context (the package name)